lab_1/main.py

```diff
@@ -36,7 +61,10 @@ def calculate_frequencies(tokens: list) -> dict:
     e.g. tokens = ['weather', 'sunny', 'man', 'happy']
     --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
     """
-    pass
+    if not isinstance(tokens, list):
+        return {}
+    else:
```

Comment:
Shorten the function by one line. Please, explain how and why it can be shortened.